keeper/component/forwarder/state.go

```
@@ -0,0 +1,183 @@
+
+	return f.pausedProtocols.Remove(ctx, int32(protocolID))
+}
+
+func (f *Forwarder) GetPausedProtocols(
+	ctx context.Context,
+) ([]core.ProtocolID, error) {
+	iter, err := f.pausedProtocols.Iterate(ctx, nil)
+	if err != nil {
+		return nil, err
+	}
+	defer iter.Close()
```

golangci-lint (2.2.2)

Error return value of `iter.Close` is not checked

(errcheck)